examples/21_create-pages/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "waku-example",
+  "name": "create-pages",
```

Comment:
I think we can use the same name for all examples. But, looking at them now, it's not consistent.

Reply:
I like giving them unique names to allow for running `pnpm -F create-pages dev` in this case. 

It would also be a nice way to reduce all the example dev scripts in the root package.json if you're ok with this

Reply:
Oh, I didn't know about `-F`. Any reference?

Examples are standalone apps. What you get with `npm create waku@latest` is the example.

Also, this should work too: https://stackblitz.com/github/dai-shi/waku/tree/v0.21.7/examples/01_template

Reply:
Nice!

And here's a reference, it's the same as --filter

https://pnpm.io/filtering

Reply:
Ah, well I often use `--filter`.

Reply:
Would you want to use it for the examples?

Reply:
Yeah, I don't like having too many `"example:*"` entries.